### ISSUE TITLE
WCAG 2.2 Violation - There are status messages that appear within the interface that are not announced to assistive technology users.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#7]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [4.1.3 Status Messages](https://www.w3.org/TR/WCAG22/#status-messages)

#### Further information:

1. 

### Instances

1. #45
